### PR TITLE
PFC RX/TX pause duration counters

### PR DESCRIPTION
**Pull request by itaibaz**

### CODE REVIEW

File: inc/saiport.h [hunk 1]
```diff
@@ -1416,6 +1416,54 @@ typedef enum _sai_port_stat_t
     /** SAI port stat PFC 7 tx pkts */
     SAI_PORT_STAT_PFC_7_TX_PKTS,
 
+    /** PFC pause duration for RX and TX per PFC priority [uint64_t] */
+    SAI_PORT_STAT_PFC_0_RX_PAUSE_DURATION,
```
Comment (lguohan) [inline inc/saiport.h]: can you explain the difference between TX versue RX pause duration?

Comment (itaibaz) [inline inc/saiport.h]: RX pause duration for certain priority is a the duration quanta in ingress pause frame for that priority (a pause frame received by the switch)
While TX pause duration for certain priority is the duration quanta in egress pause frame for that priority (a pause frame sent by the switch)

Comment (lguohan) [inline inc/saiport.h]: the description is clear to me. can you add the description in the comment?

Comment (itaibaz) [inline inc/saiport.h]: done

**Pull request merged**